lib/component_reader.js

```diff
@@ -68,12 +68,15 @@ ComponentReader.prototype.loadTriggerOrAction = function loadTriggerOrAction(nam
         result = require(modulePath);
         return Q.resolve(result);
     } catch (err) {
-        err.message = util.format(
-            "Trigger or action '%s' is not found. "
-            + "Please check if the path you specified in component.json ('%s') is valid.",
-            name,
-            filename
-        );
+        var message = "Trigger or action '%s' is found, but can not be loaded. "
+            + "Please check if the file '%s' is correct.";
+
+        if (err.code === 'MODULE_NOT_FOUND') {
```

Comment:
As far as I found in documentation – only `MODULE_NOT_FOUND` is marked by dedicated code.
Syntax error in required file leads to error without any code.